apis/filters/parse_test.go

```diff
@@ -0,0 +1,151 @@
+package filters
+
+import (
+	"testing"
+)
+
+func TestParseArgs(t *testing.T) {
+	// equivalent of `pouch ps -f 'created=today' -f 'image.name=ubuntu*' -f 'image.name=*untu'`
```

Comment:
ping @Ace-Tang This function is duplicate with your implementation about `ps filter`?